cypress/page/menu-content.page.ts

```diff
@@ -0,0 +1,13 @@
+class MenuContentPage {
+    private menuContentPageURL: string
+
+    constructor() {
+        this.menuContentPageURL = "http://automationpractice.com/"
```

Comment:
this MenuContentPage should remain as it is in the guide:
![image](https://user-images.githubusercontent.com/10209180/114577042-4f2ba600-9c41-11eb-8e71-8fde1f8fe690.png)

Reply:
Just to know, Is there any special reason to have it like the guide? Maybe further I will need the specific CSS or something?